cdc/capture/capture.go

```diff
@@ -417,6 +417,13 @@ func (c *Capture) campaignOwner(ctx cdcContext.Context) error {
 			zap.String("captureID", c.info.ID),
 			zap.Int64("ownerRev", ownerRev))
 
+		pc, err := pdutil.NewPlacementClient(ctx, c.PDClient)
+		if err != nil {
+			log.Warn("Fail to create placement client in owner", zap.Error(err))
+			return cerror.ErrCaptureSuicide.GenWithStackByArgs()
```

Comment:
1. We call this function before `setowner` to avoid setting the owner back to nil again.
2.  Return `ErrCaptureSuicide` to prevent capture from exiting abnormally.

Reply:
```suggestion
```

I prefer not to block owner initialization, it's a nice-to-have optimization.

Reply:
https://github.com/pingcap/tiflow/pull/4937/files#diff-4c39f83962361d19e42282e94c5fbd875844635ea0e9cf1db1d722eaaeace917R100-R112 
Should integration testing be removed along with it？